### PR TITLE
Linting fixes

### PR DESCRIPTION
**Pull request by lj-kpmg**

### CODE REVIEW

File: site/pages/contact.tsx [hunk 1]
```diff
@@ -13,32 +14,38 @@ const Contact = (): ReactElement => {
...
                         &nbsp;to view their contact details
                     </p>
                 </div>
                 <div className="govuk-grid-column-one-third">
                     <h2 className="govuk-heading-s">Create Transport Disruption Data Service</h2>
                     <p className="govuk-body">
-                        Create Transport Disruption Data
-                        The Create Transport Disruption Data gives local transport authorities and operators within England the ability to create disruptions for bus, light rail, ferry and tram services
-                        Help documents
+                        Create Transport Disruption Data The Create Transport Disruption Data gives local transport
```
Comment (RobCattonKPMG) [inline site/pages/contact.tsx:46]: Is this correct? Looks a little strange

File: site/pages/create-disruption.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ interface CreateDisruptionProps {
 
 const getReasonOptions = (): JSX.Element[] => {
     const options: JSX.Element[] = [
-        <option value="" disabled>
+        <option value="" disabled key="">
```
Comment (RobCattonKPMG) [inline site/pages/create-disruption.tsx:17]: Does this still correctly load with 'Choose a reason' as the default?

File: site/pages/contact.tsx (hunk 1 above)
Comment (lj-kpmg) [reply site/pages/contact.tsx:46]: Yeah there was some unnecessary text at the start there, I've removed it

File: site/pages/create-disruption.tsx (hunk 2 above)
Comment (lj-kpmg) [reply site/pages/create-disruption.tsx:17]: Yeah, a unique key is just needed by react when there's a list of elements, it still displays the same

**Review by RobCattonKPMG — APPROVED** [review]

**Pull request merged**